llama_index/utilities/sql_wrapper.py

```diff
@@ -45,9 +45,9 @@ def __init__(
         ignore_tables: Optional[List[str]] = None,
         include_tables: Optional[List[str]] = None,
         sample_rows_in_table_info: int = 3,
-        indexes_in_table_info: bool = False,
+        indexes_in_table_info: bool = True,
         custom_table_info: Optional[dict] = None,
-        view_support: bool = False,
+        view_support: bool = True,
```

Comment:
Is there some consequence for changing the default vs. Letting the dev make the decision?

Does this still work fine for normal tables without views?

Reply:
For me it does work with tables and views. I also tried a mix with tables and views. it's not a big change but I'd rather have someone else test this as well before me breaking the whole framework.